ci/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:18.04
+
+RUN ["apt-get", "update"]
```

Comment:
Please merge all of these into one `RUN` finishing each off with `&& rm -rf /var/lib/apt/lists/*` which removes the temporary state of `apt` from the image.

Reply:
Sure, done